### DISCUSSION TITLE
Personal design tweaks I would enjoy

### DISCUSSION THREAD

**Discussion by hasskyh** [lichess-org/mobile#121] [Ideas]
I’m not entirely confident on GitHub, but I came from https://www.reddit.com/r/lichess/comments/12woab2/demo_of_current_state_of_the_new_lichess_app/?utm_source=share&utm_medium=ios_app&utm_name=ioscss&utm_content=2&utm_term=1 this Reddit post, and I thought I’d write down my ideas for things I think would make the application better (sorry if this is not how this works!). 

I really like the current UI, although the squishy board is a little off putting. When it’s added, my favourite part of the website is how I can immediately open a game in one click, so I hope that will be preserved. I also miss the rating graphs for profiles. Other than that, I think it looks really good, I like the menu layout, and it seems to be very functional! I can’t wait to see the final result!

**Comment by veloce** [comment]
Not sure what you mean by "squishy" board. All board and pieces set themes have been added so you can change it.
